### ISSUE TITLE
macro with ??  gives syntax error

### ISSUE THREAD

**Issue by skelutten** [AdRoll/rebar3_format#53]
This valid macro gives syntax error with rebar format:
```erlang
-define(NASTY_MACRO(Code), {nasty, ??Code}).
```

**Comment by elbrujohalcon** [comment]
Yeah… [_stringifyied macro arguments_](https://erlang.org/doc/reference_manual/macros.html#stringifying-macro-arguments) are unparseable by either `ktn_dodger` or `epp_dodger`…

```erlang
2> rp(ktn_dodger:parse_file("/tmp/macros.erl", [{scan_opts, [text]}])).                                                                                 {ok,[{tree,attribute,                                                                                                                                              {attr,[{text,"module"},{location,1}],[],none},                                                                                                          {attribute,{tree,atom,                                                                                                                                                   {attr,[{text,"module"},{location,1}],[],none},                                                                                                          module},
                      [{tree,atom,
                             {attr,[{text,"module"},{location,1}],[],none},
                             macros}]}},
     {error,{2,erl_parse,["syntax error before: ","?"]}}]}
3> epp_dodger:parse_file("/tmp/macros.erl").
{ok,[{tree,attribute,
           {attr,1,[],none},
           {attribute,{tree,atom,{attr,1,[],none},module},
                      [{tree,atom,{attr,1,[],none},macros}]}},
     {error,{2,erl_parse,["syntax error before: ","'?'"]}}]}
```

**Comment by elbrujohalcon** [comment]
This was fixed with #207. It will look ugly until the next release of katana-code, tho.
